packages/crayons-icon/bin/index.cli.js

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env node
+
+/**
+ * @freshworks/crayons-icon
+ * Crayons-Icon as NodeJS-CLI Tool to optimize SVGs and generate JS Imports for icons.
+ *
+ * @author Freshworks <https://freshworks.in>
```

Comment:
`freshworks.in` is not the right domain :)

Reply:
Fixed